src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.TestHelpers.fsi

```diff
@@ -0,0 +1,30 @@
+module FSharp.Analyzers.SDK.TestHelpers
```

Comment:
Nitpick: I'm on the fence about words like `Helpers`.
Maybe go with `Testing` and later on with `Assert`